.github/workflows/artifacts-build.yml

```diff
@@ -58,24 +58,24 @@ jobs:
     needs: credentials
     runs-on: "ubuntu-latest"
     steps:
-    - uses: actions/checkout@v4
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```

Comment:
Did someone verify that the hashes being generated by the bot are correct?

Reply:
yep